frontend/src/pages/App/index.tsx

```diff
@@ -116,6 +117,11 @@ function App(): JSX.Element {
             path={ROUTES.EVENT_RSVP}
             render={() => <EventRsvpPage />}
           />
+          <Route
+            exact
+            path={ROUTES.EVENT_SIGN_IN_REDIRECT}
+            render={() => <EventSignInOptions />}
```

Comment:
Since our convention for naming components that are pages on the portal includes adding `Page` at the end, I think it would be great if you can change `EventSignInOptions` to `EventSignInOptionsPage`